packages/components/psammead-leading-story-promo/src/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { node } from 'prop-types';
+import Grid from '@bbc/psammead-grid';
+
+const LeadingStoryPromo = ({ image, info }) => (
+  <Grid columns={{ group4: 8 }} enableGelGutters>
```

Comment:
I think we should not be using the `Grid` within the component. This will be added in Simorgh as a wrapper, when we set up the layout that will be using this component.

Reply:
Ok